packages/web-components/src/global/variables.css

```diff
@@ -1,37 +0,0 @@
-/* Colors from: */
-/*   https://design.va.gov/design/color-palette */
-/* Fonts from: */
-/*   https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/8d39616e957cb44b6664a658b2c51dca8076fe7e/packages/formation/sass/base/_b-variables.scss#L16-L17 */
-:root {
-  --color-base: #212121;
```

Comment:
Can you document how these variables will still work in other components with the new import method you're recommending here?